### DISCUSSION TITLE
Realms?

### DISCUSSION THREAD

**Discussion by That1Rogue** [InnateAlpaca/BedrockBridge#3] [General]
Does this work for realms?

**Comment by InnateAlpaca** [comment]
Hello there! Sorry for the late reply, no, realms are not supported for now!

**Reply by LoganCaudill** [reply]
Does it work on multiplayer

**Reply by InnateAlpaca** [reply]
Hello there! If you mean on single-world then no. The addon only works on bedrock servers (bds)
